### DISCUSSION TITLE
Export query to csv failure in DBeaver v23.3.4 MacOS Sonoma Intel

### DISCUSSION THREAD

**Discussion by newscloud** [dbeaver/dbeaver#22717] [DBeaver Development]
Hi, I upgraded to v23.3.4 and tried to export a query to csv. DBeaver crashed and had to be Force Quit repeatedly.

I installed Dbeaver v23.3.0 and export query to csv worked fine.

**Comment by LonwoLonwo** [comment]
Hello @newscloud 

Looks like you have this issue: https://github.com/dbeaver/dbeaver/issues/22705

You can check the fixed version if you want: https://dbeaver.io/files/ea/
